open_academy/views/course_views.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <!-- Action of the view configuration -->
+    <record model="ir.actions.act_window" id="course_action_list">
+        <field name="name">Course view</field>
+        <field name="res_model">openacademy.course</field>
+        <field name="view_mode">tree,kanban,form</field>
+    </record>
+
+    <!-- This is tree view configuration -->
+    <record model="ir.ui.view" id="course_view_tree">
+        <field name="model">openacademy.course</field>
+        <field name="name">openacademy.course_view_tree</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="title" />
+                <field name="description"/>
+            </tree>
+        </field>
+    </record>
+
+
+    <!-- Kanban configuration -->
+    <record model="ir.ui.view" id="course_view_kanban">
```

Comment:
Kanban view is an advance view, this is not for this chapter of the documentation. Please follow the documentation.